js/packages/core/lib/dynamics.ts

```diff
@@ -44,22 +44,70 @@ export function compress(
   xn: ElemNode,
 ): NodeRepr_t;
 
-export function compress(
+export function compress(a_, b_, c_, d_, e_, f_, g_?) {
```

Comment:
It looks like this was probably a bug leftover from me, but if you don't mind fixing it while we're here I'd appreciate it–

The type signature above doesn't present an optional `props` argument up front, so we can write this function signature here as

```
export function compress(atkMs, relMs, threshold, ratio, sidechain, xn)
```

and then skip the whole `let children = (typeof` stuff below it